net/ratechecker/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2011-2017 OpenWrt.org
```

Comment:
Don't assign OpenWrt copyright unless you're affiliated, use your own or omit it.